src/elements/content-sidebar/README.md

```diff
@@ -18,6 +18,7 @@ var ContentSidebar = require('./ContentSidebar').default;
         hasMetadata
         hasSkills
         token={TOKEN}
+        {...window.ADDITIONAL_PROPS}
```

Comment:
I don't think `window` is necessary here.

Reply:
I was running into issues with other tests that don't define ADDITIONAL_PROPS, it's not defined and throws an error when used like this.